tests/src/JIT/Regression/CLR-x86-JIT/V1-M12-Beta2/b79250/b79250.il

```diff
@@ -89,7 +89,7 @@ Start_Orphan_8:
          stind.i4
 End_Orphan_8:
            ldloc LOCAL_0x0
-          conv.ovf.i1.un
+          conv.i1
```

Comment:
I followed your reasoning on why it's correct for this to throw an `OverflowException`, but I don't understand why your change exposed this.  Can you explain what was the diff in the generated code and why we weren't hitting the overflow exception before?